source/includes/User/user.md

```diff
@@ -19,7 +19,7 @@ The real-life person is represented by a Person record in OpenMRS, and a person
 > Get all non-retired Users 
```

Comment:
Get all non-retired users with the includeAll=true parameter below?

Reply:
Fixed